src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -14,31 +18,53 @@ const CONTEXT_ACTIONS = [
     // Copy to clipboard
     {
         text: 'Copy to Clipboard',
-        icon: Clipboard,
+        icon: ClipboardIcon,
+        successText: 'Copied!',
+        successIcon: Checkmark,
+
+        // If return value is true, we switch the `text` and `icon` on
+        // `ReportActionContextMenuItem` with `successText` and `successIcon` which will fallback to
+        // the `text` and `icon`
+        onPress: (action) => {
+            const lastMessage = _.last(lodashGet(action, 'message', null));
+            const html = lodashGet(lastMessage, 'html', '');
+            const text = lodashGet(lastMessage, 'text', '');
+            const isAttachment = text === '[Attachment]';
```

Comment:
No. `action.isAttachment` is not present on action.

Reply:
@marcaaron For Old attachment it's not present Is it fine to leave the old attachments?

Reply:
Hmm sorry what is an "old attachment" 😅  ?

Reply:
I mean Old messages containing attachments. :)

Reply:
Ok, why is it not present? It would be good to have one way to tell if something is an attachment and not always using this comparison to `[Attachment]` 🤔 

At a minimum we should make `[Attachment]` a constant somewhere.

Reply:
I think it was added later from the backend. I am making a utility function to check if the key is present, check that, otherwise do the text comparison.